lib/features/mailbox_dashboard/presentation/widgets/top_bar_thread_selection.dart

```diff
@@ -55,8 +55,8 @@ class TopBarThreadSelection {
       buildIconWeb(
           icon: SvgPicture.asset(
               listEmail.isAllEmailRead
-                  ? imagePaths.icRead
-                  : imagePaths.icUnread,
+                  ? imagePaths.icUnread
+                  : imagePaths.icRead,
```

Comment:
it should be better in the next time. This change should belong to other commits, because it is not related to range selection.